src/app/learn/get-started/advanced/add-entry-components/index.html

```diff
@@ -1,32 +1,32 @@
 <stache
-  pageTitle="Add Entry Components"
+  pageTitle="Entry Components"
   showTableofContents="true"
   navTitle="Entry components">
 
+  <stache-page-summary>
+    <p>
+      Entry components are components such as tiles and modals that you generate dynamically instead of with HTML selectors. You must declare entry components to ensure that Webpack does not remove them as unused code during tree-shaking.
+    </p>
+    <p>
+      You register entry components in the SPA's <stache-code>app-extras.modules.ts</stache-code> file. Entry components are not tied to routes. They are loaded dynamically and are not referenced in component templates. For more information about entry components, see the <a href="https://angular.io/guide/ngmodule-faq#what-is-an-entry-component">Angular documentation</a>.
+    </p>
+  </stache-page-summary>
+
+  <stache-page-anchor>
+    Add an entry component
+  </stache-page-anchor>
   <p>
-    Entry components are components such as tiles and modals that are generated dynamically instead of with HTML selectors. You must declare entry components to ensure that Webpack does not remove them as unused code during tree-shaking.
-  </p>
-  <p>
-    You register entry components in the <stache-code>app-extras.modules.ts</stache-code> file. Entry components are not tied to routes. They are loaded dynamically and are not referenced in component templates. For more information about entry components, see the <a href="https://angular.io/guide/ngmodule-faq#what-is-an-entry-component">Angular documentation</a>.
-  </p>
-  <p>
-    After you create a component in the default single-page application, you can create another page and add a SKY UX modal component to register an entry component.
+    The following tutorial walks you through the process to create a page with a SKY UX modal component and declare that component as an entry component.
   </p>
-
   <ol>
     <li>
       <p>
-        Return to the project in a local editor, and create a folder within the <stache-code>src/app</stache-code> folder.
+        Open your project in a local editor, and create a folder within the <stache-code>src/app</stache-code> folder. The folder name appears in the route for the page. For this tutorial, use <stache-code>modal</stache-code> for the folder name.
```

Comment:
> in the route for the page
What does "the route" refer to here? I'm not quite sure I follow this sentences intent.

Reply:
I updated the second sentence to "The folder name is represented in the page's URL."